src/sources/eventstoredb/mod.rs

```diff
@@ -0,0 +1,129 @@
+use self::types::{
```

Comment:
This module should also have the name of the source, so `src/sources/eventstoredb_metrics/mod.rs`.

Reply:
done